test/erlfmt_format_SUITE.erl

```diff
@@ -295,7 +295,126 @@ sigils(Config) when is_list(Config) ->
     ),
     ?assertSame("~s\"\"\"\n\\tabc\n\\tdef\n\"\"\"\n"),
     ?assertSame("\"\"\"\nTest\nMultiline\n\"\"\"\n"),
-    ?assertSame("~\"\"\"\nTest\nMultiline\n\"\"\"\n").
+    ?assertSame("~\"\"\"\nTest\nMultiline\n\"\"\"\n"),
+    %% Test multiline sigils in lists cause list to break
+    ?assertFormat(
+        "[~\"\n    foo\n    \"].\n",
```

Comment:
For tests, please follow the existing code style and avoid `\n` directly in strings - split them across multiple lines. This makes it possible to easier understand what's going on from looking at the test - this is very hard to understand how things are moved around.

Reply:
[d8cbea4](https://github.com/WhatsApp/erlfmt/pull/371/commits/d8cbea4f40e9f14148dcce4d08b010ef75f1e637)